features/admin.applications.v1/components/forms/advanced-configurations-form.tsx

```diff
@@ -198,125 +232,171 @@ export const AdvancedConfigurationsForm: FunctionComponent<AdvancedConfiguration
+
+                    errors.androidPackageName = t("applications:forms.advancedConfig." +
+                        "sections.platformSettings.fields." +
+                        "android.fields.androidPackageName.validations.emptyForAttestation");
+                } else if (values.androidPackageName?.toString().trim()
+                    && !Object.keys(androidAttestationServiceCredentialsObject).length) {
+
+                    errors.androidAttestationServiceCredentials = t("applications:forms." +
+                            "advancedConfig.sections.clientAttestation.fields." +
+                            "androidAttestationServiceCredentials.validations.empty");
+                }
+            } catch (ex: any) {
+                errors.androidAttestationServiceCredentials = t("applications:forms." +
+                "advancedConfig.sections.clientAttestation.fields." +
+                "androidAttestationServiceCredentials.validations.invalid");
+            }
+        } else if (values.enableFIDOTrustedApps) {
+            // Validate the android package name and android thumbprints for FIDO trusted apps.
+            // If one of them is empty throw an error.
+            if (!values.androidPackageName?.toString().trim()
+                && thumbprints?.toString().trim()) {
 
-            errors.androidPackageName = t("applications:forms.advancedConfig." +
-                "sections.applicationNativeAuthentication.fields." +
-                "android.fields.androidPackageName.validations.empty");
-        } else if (values.androidPackageName?.toString().trim()
-            && !values.androidAttestationServiceCredentials?.toString().trim()) {
+                errors.androidPackageName = t("applications:forms.advancedConfig." +
+                    "sections.platformSettings.fields." +
+                    "android.fields.androidPackageName.validations.emptyForFIDO");
+            } else if (values.androidPackageName?.toString().trim()
+                && !thumbprints?.toString().trim()) {
 
-            errors.androidAttestationServiceCredentials = t("applications:forms." +
-                "advancedConfig.sections.applicationNativeAuthentication.fields." +
-                "android.fields.androidAttestationServiceCredentials.validations.empty");
+                setShowThumbprinstError(true);
+                errors.androidThumbprints = t("applications:forms." +
+                    "advancedConfig.sections.platformSettings.fields.android.fields." +
+                    "keyHashes.validations.empty");
+            }
         }
 
         return errors;
     };
 
+    /**
+     * Handle FIDO activation value with confirmation. Deactivation is not confirmed
+     */
+    const handleFIDOActivation = (shouldActivate: boolean) => {
+        shouldActivate ?
+            setShowFIDOConfirmationModal(true)
+            : setIsFIDOTrustedAppsEnabled(false);
+    };
```

Comment:
const handleFIDOActivation = (shouldActivate: boolean) =>  setShowFIDOConfirmationModal(shouldActivate);

Reply:
This if condition uses 2 different methods, therefore cannot modify as suggested